.github/workflows/code-checks.yml

```diff
@@ -34,36 +34,17 @@ jobs:
       with:
         python-version: '3.10.6'
 
-    # Note: This uses an internal pip API and may not always work
-    # https://github.com/actions/cache/blob/master/examples.md#multiple-oss-in-a-workflow
-    - name: Cache pip
+    - name: Mypy cache
       uses: actions/cache@v3
       with:
-        path: ~/.cache/pip
-        key: pip-${{ hashFiles('requirements/pytorch/base.txt') }}
-        restore-keys: pip-
-        enableCrossOsArchive: true
+        path: .mypy_cache
+        key: mypy-${{ hashFiles('requirements/typing.txt') }}
 
     - name: Install dependencies
       run: |
-        pip install mypy==0.982  # TODO: pull this verion deom requirements/test.txt
-        pip install torch==1.12 --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
-        python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt
```

Comment:
We should still adjust versions with the proper torch version (1.12 in this case)

Reply:
The adjustment is "only" for `torchvision` and that is used in examples so not really needed for typing checks :)